boot.php

```diff
@@ -15,7 +23,12 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+if(file_exists(getcwd().'/'.$argv[1])){
+    $filename = getcwd().'/'.$argv[1];
+} else{
+    $filename = $argv[1];
```

Comment:
Sorry for this, I kept the line with `getcwd()` :sweat_smile:

Kept only the line `$filename = $argv[1]` it works fine